docs/cluster.md

```diff
@@ -166,86 +163,63 @@ To form a cluster using `DNS` strategy:
 
 1. Enable distribution mode with `JF_DIST_ENABLED=true`
 1. Chose `DNS` strategy with `JF_DIST_STRATEGY_NAME`.
+1. Set `JF_DIST_MODE` to `name`.
 1. Give your node a name with `JF_DIST_NODE_NAME`.<br /> 
-**Important** It has to be in the form of `<nodename>@<ip_address>`
+**Important** It has to be in the form of `<nodename>@<hostname>`
```

Comment:
Wouldn't it also work with `<nodename>@<ip_address>`? If not, why do we have [this if in code](https://github.com/jellyfish-dev/jellyfish/blob/6a6a95e0a6ed2abbf80a2b02c46ce48cddb16e24/lib/jellyfish/config_reader.ex#L198-L200)?

Reply:
It will, that's why I use `hostname` term here. In `JF_DIST_NODE_NAME`, `hostname` is described as something that must resolve and can be FQDN, IP, or any string depending on `JF_DIST_MODE`